cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -387,12 +398,11 @@ def _save_xblock(  # lint-amnesty, pylint: disable=too-many-statements
     publish=None,
     fields=None,
     summary_configuration_enabled=None,
-):
+):  # lint-amnesty, pylint: disable=too-many-statements
```

Comment:
if you want to pylint-disable something, do not include lint-amnesty. lint-amnesty is only for violations that were grandfathered in when we started enforcing 0 violations.
```suggestion
):  # pylint: disable=too-many-statements
```